frontend/src/common/components/ListPageNumbers.tsx

```diff
@@ -1,44 +1,36 @@
 import React from "react";
 
-import {Button} from "hds-react";
+import {Pagination} from "hds-react";
 
 import {PageInfo} from "../schemas";
 
 interface ListPageNumbersProps {
-    pageInfo: PageInfo | undefined;
+    pageInfo: PageInfo;
     currentPage: number;
     setCurrentPage: React.Dispatch<React.SetStateAction<number>>;
 }
 
-export default function ListPageNumbers({currentPage, setCurrentPage, pageInfo}: ListPageNumbersProps): JSX.Element {
-    if (pageInfo === undefined) return <></>;
-
-    const pageButtonData: {pageNumber: number; buttonText: string | number}[] = [];
-    const totalPages = pageInfo.total_pages;
-    for (let i = -3; i <= 3; i++) {
-        if (currentPage + i > 0 && currentPage + i <= totalPages) {
-            pageButtonData.push({pageNumber: currentPage + i, buttonText: currentPage + i});
-        }
-    }
-    pageButtonData.unshift({pageNumber: Math.max(1, currentPage - 1), buttonText: "<"});
-    pageButtonData.push({pageNumber: Math.min(totalPages, currentPage + 1), buttonText: ">"});
-    if (totalPages === 1) return <></>;
+export default function ListPageNumbers({
+    currentPage,
+    setCurrentPage,
+    pageInfo,
+    pageInfo: {total_pages: totalPages},
+}: ListPageNumbersProps): JSX.Element {
```

Comment:
Didn't even know this style of typing could be used 😮 
But isn't `pageInfo` typed here twice, why not just use the interface instead of retyping it like this? 🤔

Reply:
Yeah, nested abstraction. 
But no, `pageInfo` is not defined here twice. It's defined on line 16, and on line 17 we only define `totalPages`. There `pageInfo` is only given as pointer to access the `total_pages` value within it, which is then abstracted to `totalPages`. It's just a handy way to abstract the value, and as I think the ListPageNumbers was using the `totalPages` variable. I think using it within the element is better/clearer than having to refer to it e.g. as `pageInfo.total_pages`. Of course it could also be declared as a constant separately, i.e. `const totalPages = pageInfo.total_pages;` - but the abstraction is more elegant imho! 🎩